readme.adoc

```diff
@@ -29,38 +29,36 @@ Table of contents overview:
 Your feedback and contributions are most welcome! Instructions for
 building and contributing can be found in <<appendixbuild>>.
 
+=== What to expect?
+
+Bitcoin is a huge subject and we couldn't possibly cover all of its aspects here. Nevertheless, we
+hope to discuss the necessary features to get you started as well as to enable you to further explore it on your own.
+
+There are lots of people involved in Bitcoin; as some of them have opposing opinions,
+there are resources that express contradictory ideas. However, we always
+attempt to stick to the facts, where opinions do not matter.
+
+=== Who wrote this?
+
+The main authors of this book are Kalle Rosenbaum and Linnéa Rosenbaum. This work
+was commissioned and funded by Chaincode Labs, a development center that runs educational
+programs for developers who want to https://learning.chaincode.com/[learn] about Bitcoin development.
```

Comment:
> I would also add a short sentence about the authors' previous experiences and/or areas of expertise, thus giving them more authority

Yeah, that might be good. @linnearosenbaum and I talked about our roles here. We think Linnéa should be co-author. She didn't write much of it, but served an important role "behind the scenes" with review, discussions, ideas, etc.

How about this instead:

The main authors of this book is Kalle Rosenbaum and Linnéa Rosenbaum (co-author). This work
was commissioned and funded by Chaincode Labs, a development center that runs educational
programs for developers who want to https://learning.chaincode.com/[learn] about Bitcoin development.

Linnéa has https://konsensus.network/product/lilla-boken-om-bitcoin/[translated "`The Little Bitcoin Book`"] to Swedish, and sits as a board member of the Swedish Bitcoin Association. She has a solid understanding of Bitcoin, both technically and socially. She has a background in hardware development and in more recent years software development.

Kalle is the author of https://rosenbaum.se/book/["`Grokking Bitcoin`"] (Manning Publications) and is a seasoned software developer. He's been working professionally with Bitcoin-related development since 2015.

(Let's see what @linnearosenbaum has to say about the above, before moving forward with this...)

Reply:
Here's Linnéa's edit of her Bio (I missed a few things):
```
Linnéa has a Ph.D. in Electronic Systems. She is the Swedish translator of https://konsensus.network/product/lilla-boken-om-bitcoin/["`The Little Bitcoin Book`"] and co-translator of https://konsensus.network/product/sverige-bitcoinstandarden/["`The Bitcoin Standard`"].  She is  also on the board of the Swedish Bitcoin Association. Her background is in firmware development with a recent shift towards software development.
```

Reply:
Ok, I'm adding it to the text